pkg/plugins/ifaces.go

```diff
@@ -16,10 +16,10 @@ type Store interface {
 	Plugins(ctx context.Context, pluginTypes ...Type) []PluginDTO
 }
 
-type Manager interface {
-	// Add adds a plugin to the store.
+type Installer interface {
```

Comment:
Nit. Could we name these methods `InstallPlugin` and `UninstallPlugin` now? :smile: Maybe later...